### DISCUSSION TITLE
use downstream in Backstage CycloneDX Plugin

### DISCUSSION THREAD

**Discussion by regicsolutions** [CycloneDX/cyclonedx-javascript-library#215] [Show and tell]
Not sure if this is the correct forum for ideas but figured I would throw this out there given this seems like a great library to be used for a [Backstage](https://backstage.io) plugin - https://github.com/backstage/backstage/issues/13630

**Comment by jkowalleck** [comment]
Hello @regicsolutions ,

Feel free to raise issues if you need additional data models,
so we could discuss further implementations and implications.
